common/persistence/sql/sqlplugin/mysql/events.go

```diff
@@ -62,6 +62,16 @@ const (
 
 	getHistoryTreeQuery = `SELECT branch_id, data, data_encoding FROM history_tree WHERE shard_id = ? AND tree_id = ? `
 
+	paginateBranchesQuery = `SELECT shard_id,
+               tree_id,
+               branch_id,
+               data,
+               data_encoding
```

Comment:
maybe move all fields to one line? it's weird to have shard_id and the rest at different indentation levels

Reply:
fixed, thanks!